modules/update-upgrading-web.adoc

```diff
@@ -23,7 +23,7 @@ You can find information about available {product-title} advisories and updates
 link:https://access.redhat.com/downloads/content/290/ver=4.6/rhel---8/4.6.0/x86_64/product-errata[in the errata section]
 of the Customer Portal.
 ////
-update link to 4.6 when available
+update link to 4.7 when available
```

Comment:
Does the above link need to be updated to 4.7? Or is it still not available yet?

Reply:
The errata updates for 4.7 won't be available until the advisories ship (on GA). I think this link, however, can be updated now to something like:
>https://access.redhat.com/downloads/content/290/ver=4.7/rhel---8/4.7.0/x86_64/product-errata